component/header.jsx

```diff
@@ -8,6 +8,11 @@ const LinkMatch = ({ match, children, ...props }) => (
   <li>{match ? children : <a {...props}>{children}</a>}</li>
 )
 
+const LogOut = () => {
+  const onclick = () => localStorage.clear()
+  return <a href="/api/logout" onclick={onclick}>Logout</a>
+}
+
```

Comment:
I need to clear the `localStorage` when we click on it.

I kept a real link and did not do the action through `fetch` because we want the redirection to clear the cookie.
Since it's a secure cookie it's not possible to remove it from JS, we need to have a page refresh here.

I think it's ok for the case of the logout to do it.